index.js

```diff
@@ -2061,12 +2067,12 @@
   //. Multiple constraints may be placed on a type variable by including
   //. multiple `TypeClass` values in the array (e.g. `{a: [Foo, Bar, Baz]}`).
 
-  //  invalidArgumentsCount :: (TypeInfo, Integer, Integer, Array Any) -> Error
+  //  invalidArgumentsCount :: (TypeInfo, Integer, Integer, Array Any, Function) -> Error
```

Comment:
Note: This line is too long now. Argument types should be moved into the argument list as is done with other functions.